kibana.md

```diff
@@ -181,11 +181,33 @@ $ bin/kibana-plugin remove readonlyrest_kbn
 To upgrade to a new version of a ReadonlyREST plugins for Kibana, you should:
 
 * [Unpatch Kibana](./#unpatching-kibana)
-* [Uninstall](./details/kibana-7.8.x-and-older.md#uninstall) the old plugin
-* Delete all the content of "optimize" directory (in the main Kibana installation directory) `rm -rf optimize/`
+* [Uninstall](#uninstalling) the old plugin
 * [Install](./#installation) the new one
 * [Patch Kibana](./#patching-kibana)
-* Restart Kibana.
+* Restart Kibana
+
+### Major version upgrades when using multi-tenancy
+If you use multi-tenancy (Enterprise only), you will have one or more teanancy-specific kibana indices beyond the main `.kibana` (e.g. `.kibana_tenant1`, `.kibana_tenant2`, etc.).
+
+The first time you run Kibana after a major version upgrade (e.g. upgrading from Kibana 7.17.7 to Kibana 8.0.0), Kibana will run a [saved objects migration](https://www.elastic.co/guide/en/kibana/current/saved-object-migrations.html) on the default `.kibana` index, or whatever it finds configured as `kibana.index` in `kibana.yml`.
+
+Now, because you may have multiple kibana indices containing saved objects, you should apply the "saved object migration" to those indices as well.
+
+The easiest way to do this is also the most "ignorant" one: 
+
+**Preparation**
+
+* download the newer Kibana version in your laptop (in the example, 8.0.0)
```

Comment:
`in your laptop` seems too specific maybe `to your computer` will be better here 🤔 ?